onnxruntime/core/framework/onnxruntime_typeinfo.cc

```diff
@@ -6,14 +6,18 @@
 #include <cassert>
 #include "onnxruntime_typeinfo.h"
 #include "core/framework/tensor.h"
+#include "core/framework/sparse_tensor.h"
 #include "core/graph/onnx_protobuf.h"
 
 using onnxruntime::BFloat16;
 using onnxruntime::DataTypeImpl;
 using onnxruntime::MLFloat16;
 using onnxruntime::Tensor;
+using onnxruntime::SparseTensor;
 using onnxruntime::TensorShape;
 
+namespace on = ONNX_NAMESPACE;
```

Comment:
Is this used anywhere else? Would it be better to just use the namespace?

Reply:
The macro is needed due to flexibility but it is too long to type in switch() cases so I abbreviated it. This is not usable outside this file.

---
In reply to: [321456846](https://github.com/microsoft/onnxruntime/pull/1714#discussion_r321456846) [](ancestors = 321456846)